### PR TITLE
[Reply] Cap body at 1340 for desktop and tablet

### LINKED ISSUE #298: [Reply] Cap the desktop body width to 1340
Currently, the width extends as wide as it can be with constant margins, but that body with should be capped at 1340 for better text readability.

### PR DESCRIPTION
**Pull request by Renzo-Olivares**
This change caps the body of the reply material study to 1340 for better text readability.

desktop_cap

Closes #298

### CODE REVIEW

File: lib/studies/reply/adaptive_nav.dart [hunk 1]
```diff
@@ -241,12 +241,23 @@ class _DesktopNavState extends State<_DesktopNav>
...
             },
           ),
-          const VerticalDivider(thickness: 1, width: 1),
+          const VerticalDivider(thickness: 1, width: 2),
           Expanded(
-            child: _SharedAxisTransitionSwitcher(
-              defaultChild: _MailNavigator(
-                child: MailboxBody(key: widget.inboxKey),
-              ),
+            child: Row(
```
Comment (rami-a) [inline lib/studies/reply/adaptive_nav.dart:246]: I think you should be able to achieve this without using a Row. Perhaps replace the Row and Flexible widgets with a Center widget?

File: lib/studies/reply/adaptive_nav.dart [hunk 2]
```diff
@@ -241,12 +241,23 @@ class _DesktopNavState extends State<_DesktopNav>
               );
             },
           ),
-          const VerticalDivider(thickness: 1, width: 1),
+          const VerticalDivider(thickness: 1, width: 2),
```
Comment (rami-a) [inline lib/studies/reply/adaptive_nav.dart:244]: Whats the reason for this change?

**Review by rami-a — COMMENTED** [review]
Thanks @Renzo-Olivares 

Could you please take a look at the comments, and also update this branch against the latest master as I have fixed broken tests

File: lib/studies/reply/adaptive_nav.dart (hunk 1 above)
Comment (Renzo-Olivares) [reply lib/studies/reply/adaptive_nav.dart:246]: Thanks for the suggestion that works!

File: lib/studies/reply/adaptive_nav.dart (hunk 2 above)
Comment (Renzo-Olivares) [reply lib/studies/reply/adaptive_nav.dart:244]: white_lines

I increased the width to remedy some artifacts (white lines shown above). From my observation, they seem to be a result of the navigation rail body not expanding to its max-width like it previously did before this change.

**Review by rami-a — APPROVED** [review]
LGTM

**Comment by clocksmith** [comment]
LGTM, assuming there are still constant margins for when the width is less than 1340

File: lib/studies/reply/adaptive_nav.dart (hunk 2 above)
Comment (Renzo-Olivares) [reply lib/studies/reply/adaptive_nav.dart:244]: These lines still show up if I don't have my browser window maximized. I'll look into it some more.

File: lib/studies/reply/adaptive_nav.dart [hunk 3]
```diff
@@ -241,11 +241,17 @@ class _DesktopNavState extends State<_DesktopNav>
...
           Expanded(
-            child: _SharedAxisTransitionSwitcher(
-              defaultChild: _MailNavigator(
-                child: MailboxBody(key: widget.inboxKey),
+            child: Center(
+              child: ConstrainedBox(
+                constraints:
+                    BoxConstraints(maxWidth: widget.extended ? 1340 : 800),
```
[the remark this answers is not in the document]
Comment (Renzo-Olivares) [reply lib/studies/reply/adaptive_nav.dart]: Yeah, you're right. I misunderstood the purpose of the cap and made it consistent for both desktop and tablet. I changed it back to match the original issue, also the margins were not constant when the width was under 1340 with the added 800 cap.

**Pull request merged**